INSTALL.md

```diff
@@ -1,108 +1,92 @@
 # PyBitmessage Installation Instructions
 
-For an up-to-date version of these instructions, please visit the
-[Bitmessage Wiki](https://bitmessage.org/wiki/Compiling_instructions).
+## Download
+Download for Windows (32bit) (64bit)
+https://github.com/Bitmessage/PyBitmessage/releases/download/v0.6.1/Bitmessage-0.6.1.exe
 
-PyBitmessage can be run in either one of two ways:
+Download for Apple OS X (not up to date)
+https://github.com/Bitmessage/PyBitmessage/releases/download/v0.6.1/bitmessage-v0.6.1.dmg
 
-- straight from source
-
- or
-- from an installed
-package.
+For Linux follow below given steps & for detailed instruction check [Bitmessage Wiki](https://bitmessage.org/wiki/Compiling_instructions).
+You must download the source
+```
+git clone git://github.com/Bitmessage/PyBitmessage.git
+```
 
 ## Dependencies
 Before running PyBitmessage, make sure you have all the necessary dependencies
 installed on your system.
 
-Here's a list of dependencies needed for PyBitmessage
-- python2.7
-- python2-qt4 (python-qt4 on Debian/Ubuntu)
-- openssl
-- (Fedora & Redhat only) openssl-compat-bitcoin-libs
-
-## Running PyBitmessage
-PyBitmessage can be run in two ways:
-- straight from source
-
- or
-- via a package which is installed on your system. Since PyBitmessage is Beta, it is best to run
-PyBitmessage from source, so that you may update as needed.
+Here's a list of dependencies needed for PyBitmessage based on operating system
```

Comment:
these dependencies may not be up to date and PyBitmessage may not build on a recent OS version